website/docs/docs/cloud/about-cloud/single-tenant.md

```diff
@@ -13,5 +13,5 @@ _To learn more about setting up a dbt Cloud Single Tenant deployment, [please co
 
 For more information about the dbt Cloud Single Tenant deployment see the below.
```

Comment:
is this sentence and the links below it still needed? or should it just link to dbt architecture?

Reply:
Removed the entire page. Thanks for catching!